src/cascadia/TerminalSettingsEditor/Appearances.idl

```diff
@@ -49,8 +45,12 @@ namespace Microsoft.Terminal.Settings.Editor
     {
         Appearances();
         AppearanceViewModel Appearance;
+        ProfileViewModel SourceProfile;
         static Windows.UI.Xaml.DependencyProperty AppearanceProperty { get; };
 
+        Boolean UsingMonospaceFont { get; };
+        Boolean ShowAllFonts;
+
```

Comment:
Wait... Wouldn't we still want `UsingMonospaceFont` and `ShowAllFonts` in the `AppearanceViewModel`?

Here's my reasoning. When we have multiple appearances, we'll have one `AppearanceViewModel` for each of them. Each view (like, in the UI) will have it's own configuration, so we need to independently track these two properties for each view model.

Reply:
Different AppearanceConfigs/AppearanceViewModels don't have their own fonts though (we don't allow changing anything to do with font) - only the profile's font can be changed (we moved the font things into the AppearanceViewModel anyway just so we could maintain the 'Text' grouping in the SUI, but the font things will not be visible for any appearance other than default)

Reply:
Omg, that's a good point. My bad!